modules/distr-tracing-rn-new-features.adoc

```diff
@@ -0,0 +1,40 @@
+////
+Module included in the following assemblies:
+* distr-tracing--release-notes.adoc
+////
+////
+Feature – Describe the new functionality available to the customer. For enhancements, try to describe as specifically as possible where the customer will see changes.
+Reason – If known, include why has the enhancement been implemented (use case, performance, technology, etc.). For example, showcases integration of X with Y, demonstrates Z API feature, includes latest framework bug fixes.
+Result – If changed, describe the current user experience.
+////
+
+[id="distr-tracing-rn-new-features_{context}"]
+== New features and enhancements {ProductName} 2.0.0
+
+This release marks the rebranding of Red Hat OpenShift Jaeger to {ProductName}.  This effort includes the following:
+
+* An updated {ProductShortName} Operator based on Jaeger 1.28.
```

Comment:
I'm not as familiar with writing new features, but I'm used to bullet lists being consistent. 

So, like a bullet list would consist of all nouns, `an updated distributed tracing Operator` , `A new OpenTelemetry Operator` etc. 

Or consists of all verbs, `Adds support for OpenTelemetry protocol` , `updates to the documentation` 

Or consists of all statements `There is a new distributed tracing icon` , `Going forward, Red Hat OpenShift distributed tracing will only support...` etc. 

Can we make this bullet list consistent? I think it will be easier to read.

Reply:
Revised this list.